.clang-tidy

```diff
@@ -0,0 +1,5 @@
+Checks: '-*,readability-identifier-naming'
+CheckOptions:
+- key:             readability-identifier-naming.PrivateMemberPrefix
+  value:           m_
+WarningsAsErrors: '*'
```

Comment:
As stated before, I would suggest to put the header filter into this file.
This just helps, if I want to run clang-tidy manually (maybe just on a single file...)

Reply:
`Error parsing /home/bond/Documents/GitHub/cuberite/.clang-tidy: Invalid argument
YAML:6:15: error: unknown key 'HeaderFilter'
HeaderFilter: '.+/cuberite/src/[^\.].+'`
clang-tidy doesn't like it, or I don't know the YAML syntax good enough.

Reply:
`HeaderFilterRegex: '.+/cuberite/src/[^\.].+'`
Now it run clang-tidy with this as header filter: `^/home/bond/Documents/GitHub/cuberite/tidy-build/.*`

Reply:
Ok, thanks. Yes, `IniFile.h` etc are checked, I don't know why, but they are.